dotnet/docs/api/class-page.mdx

```diff
@@ -942,7 +942,7 @@ Returns whether the element is [visible](./actionability.mdx#visible). `selector
 
 The page's main frame. Page is guaranteed to have a main frame which persists during navigations.
 
-## Page.OpenerAsync()
```

Comment:
I thought we decided to keep it async in typescript for backwards compatibility and just be consistent in C# and have it async too. Did it change?

Reply:
Oh, I did not know about the agreement :/